src/levels.json

```diff
@@ -0,0 +1,180 @@
+[
+  {
+    level_prize: 'path/to/img_cactus',
+    stages: [
+      {
+        complexity: 1,
+        type: 'primer_open',
```

Comment:
need to mix 3 types at least - 1 with primer open, 1 with primer closed, 1 with zadacha closed